### DISCUSSION TITLE
use of other Seurat fxns ['FindConservedMarkers( )', 'FindAllMarkers(  )', and 'subset(  )'] with scATACseq data in Signac

### DISCUSSION THREAD

**Discussion by BartBryant** [stuart-lab/signac#1672] [General]
While we can use 'FindMarkers(   )' to query across 2 cell types with 'ident.1' and 'ident.2',
can
we use 'FindConservedMarkers(   )' and/or 'FindAllMarkers(   )' with the sparse scATACseq dataset to find 'markers' that are unique to the cell type of interest across all cell types in the data set?

if we can, does the output follow Seurat's scRNAseq tutorials? Given sparsity of scATACseq data, would assume these functions would not work easily 

additionally, can we use 'subset(   )' to subset/filter cell types of interest to enable one to query cell type of interest without other cell types? 

thanks to all for any input/questions to this topic
i can elaborate more if my topics seem to broad

thx, 
barT

**Comment by timoast** [comment]
> we use 'FindConservedMarkers( )' and/or 'FindAllMarkers( )' with the sparse scATACseq dataset to find 'markers' that are unique to the cell type of interest across all cell types in the data set?
> additionally, can we use 'subset( )' to subset/filter cell types of interest to enable one to query cell type of interest without other cell types?

Yes, in general you can use any of the Seurat functions since Signac uses the Seurat object.
